components/atom/popover/src/index.js

```diff
@@ -91,6 +90,8 @@ AtomPopover.propTypes = {
   content: PropTypes.element.isRequired,
   /** Popover children */
   children: PropTypes.node.isRequired,
+  /** Initial value for the show pop over */
```

Comment:
Instead initial, should be something like `Controlled value for the showPopover state`.